src/ProgressOnderwijsUtils/Data/ColumnDefinition.cs

```diff
@@ -52,4 +53,7 @@ static ColumnAccessibility DbColumnMetaDataAccessibility(DbColumnMetaData col)
             { HasDefaultValue : true, } => ColumnAccessibility.NormalWithDefaultValue,
             _ => ColumnAccessibility.Normal,
         };
+
+    public static bool ShouldIncludePropertyInSqlInsert(IPocoProperty o)
+        => o.CanRead && o.PropertyInfo.Attr<DatabaseGeneratedAttribute>() is not { DatabaseGeneratedOption: not DatabaseGeneratedOption.None, };
```

Comment:
dubbele ontkenning?

Reply:
yep, want het is DB-generated als het attrs _aanwezig_ en _not-none_ is.